test/io-tests/test_io.py

```diff
@@ -578,16 +593,15 @@ def test_db_schema_reload(tmp_path, defaultenv):
     configfile = tmp_path / "test.config"
     configfile.write_text(config)
 
-    headers = {"Accept-Profile": "v1"}
     env = {key: value for key, value in defaultenv.items() if key != "PGRST_DB_SCHEMAS"}
 
     with run(configfile, env=env) as postgrest:
-        response = postgrest.session.get("/parents", headers=headers)
-        assert response.status_code == 404
+        response = postgrest.session.get("/rpc/get_guc_value?name=search_path")
+        assert response.text == '"public, public"'
```

Comment:
Instead of testing the changed db-schema implicitly, I opted to test it explicitly by looking at the search path. It didn't make sense to me to copy over the `/parents` table just for that kind of test, when we can instead do it like this.